release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -120,7 +120,40 @@ For more information, see xref:../networking/ovn_kubernetes_network_provider/tra
 
 [id="ocp-4-8-storage"]
 === Storage
+[id="ocp-4-8-storage-gcp-pd-csi-ga"]
+==== Persistent storage using GCP PD Container Storage Interface (CSI) driver operator is generally available
+The Google Cloud Platform (GCP) persistent disk (PD) CSI driver is automatically deployed and managed on GCP environments, allowing you to dynamically provision these volumes without having to install the driver manually.
+This feature was previously introduced as a Technology Preview feature in {product-title}  4.7 and is now generally available and enabled by default in {product-title} 4.8.
 
+For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html-single/storage/#persistent-storage-csi-gcp-pd[GCP PD CSI Driver Operator].
+
+[id="ocp-4-8-storage-azure-csi-tp"]
+==== Persistent storage using the Azure Disk CSI Driver Operator (Technology Preview)
+The _Azure Disk Container Storage Interface (CSI) Driver Operator_ provides a StorageClass by default that you can use to create persistent volume claims (PVCs).
+The Azure Disk Driver Operator that manages this driver is in Technology Preview.
+
+For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html-single/storage/#persistent-storage-csi-azure[Azure Disk CSI Driver Operator].
+
+[id="ocp-4-8-storage-vsphere-csi-tp"]
+==== Persistent storage using the vSphere CSI Driver Operator (Technology Preview)
+The _vSphere Container Storage Interface (CSI) Driver Operator_ provides a StorageClass by default that you can use to create persistent volume claims (PVCs).
+The _vSphere Driver Operator_ that manages this driver is in Technology Preview.
+
+For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html-single/storage/#persistent-storage-csi-vsphere[vSphere CSI Driver Operator].
+
+[id="ocp-4-8-storage-csi-migration"]
+==== Automatic CSI migration (Technology Preview)
+Starting with {product-title} 4.8, automatic migration for the following in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers is available as a Technology Preview (TP) feature:
```

Comment:
You don't need `(TP)` here. I don't think we refer to the initialism externally in the docs.